tools/generator/src/Extensions/PBXProductType+Extensions.swift

```diff
@@ -292,4 +292,8 @@ extension PBXProductType {
             return true
         }
     }
+
+    var isTopLevel: Bool {
+        return isApplication || isTestBundle
```

Comment:
This doesn't include command line tools. Should it be `isLaunchable || isTestBundle`?

Reply:
Good catch. I will update it.